examples/pytorch/rgcn/README.md

```diff
@@ -28,23 +28,22 @@ For mini-batch training, run with the following (available datasets are the same
 ```bash
 python3 entity_sample.py --dataset aifb
 ```
-For multi-gpu training (with sampling), run with the following (same datasets and GPU IDs separated by comma) 
+For multi-gpu training (with sampling), run with the following (same datasets and GPU IDs separated by comma)
 ```bash
 python3 entity_sample_multi_gpu.py --dataset aifb --gpu 0,1
 ```
 
-
 ### Link Prediction
-FB15k-237 in RAW-MRR
-```
-python link.py --gpu 0 --eval-protocol raw
-```
-FB15k-237 in Filtered-MRR
-```
-python link.py --gpu 0 --eval-protocol filtered
+
+Run with the following for link prediction on dataset FB15k-237 with filtered-MRR
+
+```bash
+python link.py
```

Comment:
Do we still support raw MRR then?

Reply:
No. It only works with filtered MRR. 
Do you want to keep the option `--eval-protocol` and add raw MRR back?

Reply:
I'm ok with keeping one option. However, I'd like to know if there are any reasons for your preference of filtered MRR over raw MRR.